### PR TITLE
GLT-801 Paginate List Pools page

### PR DESCRIPTION
**Pull request by callunity**

### CODE REVIEW

File: miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/PoolRestController.java [hunk 1]
```diff
@@ -71,7 +82,16 @@ public void setRequestManager(RequestManager requestManager) {
     this.requestManager = requestManager;
   }
 
-  @RequestMapping(value = "{poolId}", method = RequestMethod.GET, produces="application/json")
+  public PoolDto writeUrls(PoolDto poolDto, UriComponentsBuilder uriBuilder) throws IOException {
```
Comment (apmasell) [inline miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/PoolRestController.java]: Could this go in the DTO like it now is for the `SampleDto`?

**Comment by apmasell** [comment]
:+1:

File: miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/PoolRestController.java (hunk 1 above)
Comment (callunity) [inline miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/PoolRestController.java]: I don't see why not.

**Comment by djcooke** [comment]
:+1:

**Pull request merged**